.github/workflows/production.yml

```
@@ -0,0 +1,75 @@
+        username: ${{ env.DOCKER_USERNAME }}
+        password: ${{ env.DOCKER_PASSWORD }}
+    
+    - name: Pull staging image
+      run: |
+        docker pull gabcat/merchtrack:staging
+    
+    - name: Tag image for production
+      run: |
+        docker tag gabcat/merchtrack:staging gabcat/merchtrack:${{ github.event.inputs.commit_tag }}
+        docker tag gabcat/merchtrack:staging gabcat/merchtrack:latest
+    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)